src/tabs/tabs.scss

```diff
@@ -4,100 +4,103 @@
 
 @mixin iui-tabs {
   @include iui-reset;
-
-  @include iui-font-family;
   display: flex;
+  align-items: center;
   list-style: none;
   padding: 0;
-  margin: round($iui-baseline/2) 0 0 0;
   user-select: none;
 
   > li {
-    display: inline-block;
     position: relative;
```

Comment:
Is this needed?

Reply:
Previously, the `a` was positioned absolutely but now I think it can be removed

Reply:
I can move `position: relative;` off of the `<li>` and instead apply it to the `.iui-tab`.  Without it, the `::after` part does this.
![Screen Shot 2021-06-16 at 4 46 52 PM](https://user-images.githubusercontent.com/849817/122291129-76615680-cec2-11eb-9e2f-1eff4a009e84.png)

If I end up reverting the `::after` and going with `box-shadow` like originally I do think this can be removed then.